integration-tests/test/system-addresses.spec.ts

```diff
@@ -0,0 +1,107 @@
+/* Imports: External */
+import { BigNumber, Contract, ContractFactory, utils, Wallet } from 'ethers'
+import { ethers } from 'hardhat'
+
+/* Imports: Internal */
+import { expect } from './shared/setup'
+import { OptimismEnv } from './shared/env'
+import { envConfig } from './shared/utils'
+
+const SYSTEM_ADDRESSES = [
+  '0x4200000000000000000000000000000000000042',
```

Comment:
Perhaps these could be included in the predeploys object in the contracts package? That is where the rest of them are defined